### PR TITLE
Test rust-code-analysis on stable

### PR DESCRIPTION
**Pull request by Luni-4**
Test `rust-code-analysis` on `Linux` using the Rust stable version

### CODE REVIEW

File: .taskcluster.yml [hunk 1]
```diff
@@ -41,7 +41,6 @@ tasks:
             - "-cx"
             - "curl https://bootstrap.pypa.io/get-pip.py -o get-pip.py && python3 get-pip.py &&
                rustup toolchain install stable &&
-               rustup toolchain install nightly &&
                rustup default stable &&
```
Comment (marco-c) [inline .taskcluster.yml:44]: I wonder if this line is still needed.

Comment (Luni-4) [reply .taskcluster.yml:44]: Since tests are passing it's not needed anymore :)

Comment (marco-c) [reply .taskcluster.yml:44]: Well, the line is still there :P

Comment (Luni-4) [reply .taskcluster.yml:44]: Ouch, you're right!

**Pull request merged**